src/tests/ftest/pool/attribute.py

```diff
@@ -154,21 +158,23 @@ def test_pool_attributes(self):
         name = self.params.get("name", '/run/attrtests/name_handles/*/')
         expected_for_param.append(name[1])
         value = self.params.get("value", '/run/attrtests/value_handles/*/')
-        if value[0] is None:
-            self.cancel("skipping these tests until DAOS-2170 is fixed")
         expected_for_param.append(value[1])
 
+        expected_result = 'PASS'
+        for result in expected_for_param:
+            if result == 'FAIL':
+                expected_result = 'FAIL'
+                break
+
         attr_dict = {name[0]:value[0]}
         if name[0] is not None:
             if "largenumberofattr" in name[0]:
                 self.create_data_set()
                 attr_dict = self.large_data_set
+                #ramdonly generated data result is unpredictable
+                #let the callback function provide the result
+                expected_result = 'TBD'
```

Comment:
what callback function?  TBD would be unexpected in the rest of this function.  If create_data_set returns unsupported attribute data then we need to change that function.